### PR TITLE
Section 12.2.1.1: RTCErrorDetailType Enum definition

### PR DESCRIPTION
**Pull request by aboba**
Work-in-progress (do not merge). 

Fix for Issue https://github.com/w3c/webrtc-pc/issues/1044

### CODE REVIEW

**Comment by fluffy** [comment]
LGTM

File: webrtc.html [hunk 1]
```diff
@@ -11573,9 +11573,89 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
...
+              "idp-bad-script-failure",
+              "idp-execution-failure",
+              "idp-load-failure",
+              "idp-need-login",
+              "idp-timeout",
+              "idp-tls-failure",
+              "idp-token-expired",
+              "idp-token-invalid",
+              "parameters-error",
+              "sctp-failure",
```
Comment (taylor-b) [inline webrtc.html]: Some of these (sctp-failure, parameters-error, dtls-failure, data-channel-failure) aren't used yet. May want to at least make a note of that, so we don't accidentally end up with an enum value that's not used anywhere.

Comment (aboba) [inline webrtc.html]: Added a comment for parameters-error and dtls-failure.  sctp-failure and data-channel-failure are defined in PR https://github.com/w3c/webrtc-pc/pull/1011 .

File: webrtc.html [hunk 2]
```diff
@@ -11573,9 +11573,89 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
...
+                <td>The <code>RTCRtpParameters</code> is not valid.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>sctp-failure</code></dfn></td>
+                <td>The SCTP negotiation has failed or the connection
+                  has been terminated with a fatal error.</td>
+              </tr>
+              <tr>
+                <td><dfn><code>sdp-syntax-error</code></dfn></td>
+                <td>The SDP syntax is not valid.</td>
```
Comment (aboba) [inline webrtc.html]: Added material about sdpLineNumber in "sdp-syntax-error", sctpCauseCode in "sctp-failure", httpRequestStatusCode in "idp-load-failure" and idpLoginUrl in "idp-need-login".

**Pull request merged**